app/assets/stylesheets/blacklight.scss

```diff
@@ -73,8 +73,26 @@ $remove-icon-font-family:
 }
 
 .range_limit {
+  input {
+    width: 14ch !important;
+  }
+
+  .input-group {
+    display: flex !important;
+    justify-content: center;
+  }
+
+  // Blacklight 6's range limit didn't have this button
+  .view_larger {
+    display: none;
+  }
+
+  .input-group-append {
+    display: flex;
+  }
+
   .submit {
-    width: 14ch;
+    @extend .btn-sm;
```

Comment:
Can we avoid using `@extend` so that we can decouple from SASS?

One alternative is to set:
```css
  --bs-btn-padding-y: 0.25rem;
  --bs-btn-padding-x: 0.5rem;
  --bs-btn-font-size: 0.875rem;
  --bs-btn-border-radius: var(--bs-border-radius-sm);
```